prbt_hardware_support/test/unittests/unittest_brake_test_executor.cpp

```diff
@@ -47,6 +49,31 @@ class SystemMock
 
 };
 
+
+
+TEST(BrakeTestExecutorTest, testAddressSanitizing)
```

Comment:
Just for clarification: This will be dropped finally?

Reply:
Of course. Just for testing.